hangman.php

```diff
@@ -0,0 +1,88 @@
+<?php
+    $word="";
+    function getword()
+    {
+        $file="BandsList.txt";
+        $fopen = fopen($file, "r");
+        $fread = fread($fopen,filesize($file));
+        $split = explode("\n", $fread);
+        $array[] = null;
+        foreach ($split as $string)
+        {
+            array_push($array,$string);
+        }
+        $word_index = array_rand($array,1);
```

Comment:
I've asked you all to keep the PHP variable and function names camelCased.
Also the DB name, DB columns and table names underscore_separated.